app/views/DeclarantContactDetailsView.scala.html

```diff
@@ -56,7 +59,8 @@
             form = form,
             id = "phoneNumber",
             name = "phoneNumber",
-            label = "UK telephone number",
+            label = "Telephone number",
```

Comment:
The content for this label should come from the messages file as should the labels for email address and name. I realise this is not necessarily a part of this story but can you update these also as it'll make life much simpler once we introduce Welsh translations in a few sprints?

Reply:
sure, no problem